pulse_actions/handlers/talos.py

```diff
@@ -0,0 +1,79 @@
+"""
+This module is for the following use case:
+
+ - Talos jobs:
+
+    * Trigger talos jobs twice if they are from PGO build.
+"""
+import logging
+
+from pulse_actions.utils.misc import filter_invalid_builders
+
+from mozci import query_jobs
+from mozci.errors import PushlogError
+from mozci.mozci import (
+    trigger_job
+)
+from mozci.sources import buildjson
+from requests.exceptions import ConnectionError
+
+LOG = logging.getLogger()
+
+def on_event(data, message, dry_run):
+    """Trigger talos jobs twice in PGO builds."""
+    # Cleaning mozci caches
+    buildjson.BUILDS_CACHE = {}
+    query_jobs.JOBS_CACHE = {}
+    payload = data["payload"]
+    status = payload["status"]
+    buildername = payload["buildername"]
+
+    if "pgo" in buildername and ("mozilla-inbound" in buildername or "fx-team" in buildername):
+        buildername = filter_invalid_builders(buildername)
+
+        # Treeherder can send us invalid builder names
+        # https://bugzilla.mozilla.org/show_bug.cgi?id=1242038
+        if buildername is None:
+            if not dry_run:
+                # We need to ack the message to remove it from our queue
+                message.ack()
+            return
+
+        revision = payload["revision"]
+        LOG.info("**")  # visual separator
+        LOG.info("Failed job found at revision %s. Buildername: %s",
```

Comment:
How do you know this is a failed job? You don't check status.

In any case, talos jobs rarely fail. They don't turn orange like test jobs since they're testing performance and we don't change the colour of a job for performance.